### ISSUE TITLE
Can I c# programming in this application ?

### ISSUE THREAD

**Issue by Don1L** [thebaselab/codeapp#538]
Hello. I want to buy this application but I have a couple of questions. Is it possible to program normally in c# in this application and will it be possible to use the code written here to work later with it in Unity

**Comment by CodeStrate** [comment]
Yes there's C# too but it's Remote Compiled i.e. you need Internet connection. Unlike local compiled languages.
